app-modules/portal/src/Settings/PortalSettings.php

```diff
@@ -65,6 +65,8 @@ class PortalSettings extends SettingsWithMedia
 
     public string $gdpr_banner_text = "We use cookies to personalize content, to provide social media features, and to analyze our traffic. We also share information about your use of our site with our partners who may combine it with other information that you've provided to them or that they've collected from your use of their services.";
 
+    public array $cookie_gdpr_banner_text = [];
```

Comment:
Can you explain the reasoning behind a new property rather than modifying the existing one? Is it possible to use the same one? Perhaps by temporarily changing the type hint to be `string|array`.

Also is it possible to make the default for this one here include the content?